clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -963,13 +963,6 @@ static Expected<StringRef> linkDevice(ArrayRef<StringRef> InputFiles,
   SmallVector<StringRef, 16> InputFilesVec;
   for (StringRef InputFile : InputFiles)
     InputFilesVec.emplace_back(InputFile);
-  // First llvm-link step.
-  auto LinkedFile = sycl::linkDeviceInputFiles(InputFilesVec, Args);
```

Comment:
Did you test a few basic examples to make sure the results execute successfully on a GPU?

Reply:
Not exactly. Let me do that. I will keep this PR in draft mode till I can get to verify this. Thanks